tools/licensescan/main.go

```diff
@@ -35,6 +36,9 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
+//go:embed modules/*
+var modulesFS embed.FS
```

Comment:
interesting: eventually we can have it in its own Go package that will make this DB portable across all the tools :)

Reply:
I hope so :-)